openmls/src/treesync/mod.rs

```diff
@@ -336,6 +348,24 @@ impl TreeSync {
         Ok(leaves)
     }
 
+    /// Returns a [`BTreeMap`] mapping [`KeyPackageRef`]s to the corresponding
+    /// [`KeyPackage`] instances in the leaves. The map only contains full
+    /// nodes.
+    pub(crate) fn leaves(
```

Comment:
There's now a `leaves` and a `full_leaves` function, where the difference is that `full_leaves` returns a map with `LeafIndex`. This can probably be cleaned up a bit.

Reply:
Yes, `full_leaves` should probably go away.
But I didn't want to change even more code. This PR really only touches what's necessary to make it work 😬. I filed #732 for this.